routing.rst

```diff
@@ -998,7 +991,7 @@ controller action. Instead of ``string $slug``, add ``BlogPost $post``::
     {
         // ...
 
-        #[Route('/blog/{slug}', name: 'blog_show')]
+        #[Route('/blog/{slug:post}', name: 'blog_show')]
```

Comment:
the example is using deprecated implicit mapping that doesn't work anymore with the default recipes since 7.1